pkg/traceparser/parser.go

```diff
@@ -268,9 +272,16 @@ func (tp *traceParser) requestSpanStart() *tracepb2.SpanStart {
 				RequestPayload:   tp.ByteString(),
 				ExtCorrelationId: ptrOrNil(tp.String()),
 				Uid:              ptrOrNil(tp.String()),
+				Mocked:           false, // introduced in version 101
```

Comment:
Nicer to do it inline: `(func() bool { ... })()`; that way we don't need the type assertion below